libraries/Adapters/Microsoft.Bot.Builder.Adapters.WeChat/Storage/AccessTokenStorage.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Bot.Builder.Adapters.WeChat.Schema;
+
+namespace Microsoft.Bot.Builder.Adapters.WeChat
```

Comment:
Namespaces should match folder name, rename to Microsoft.Bot.Builder.Adapters.WeChat.Storage